### PR TITLE
feat: price requests v2 updates

### PR DESCRIPTION
**Pull request by md0x**
Changes proposed in this PR:

- Index user stake/unstake information
- Index staking rewards updates and rewards claims
- Index slashing information
- Updated VotingV2 contract in goerli
- Removed unused data

Latest subgraph available here:
https://thegraph.com/hosted-service/subgraph/md0x/votingv2-version-24

### CODE REVIEW

File: packages/votingV2/src/mappings/votingV2.ts [hunk 1]
```diff
@@ -38,10 +44,19 @@ export function handlePriceRequestAdded(event: PriceRequestAdded): void {
...
   request.identifier = event.params.identifier.toString();
+  request.requestIndex = event.params.requestIndex;
   request.latestRound = requestRound.id;
   request.time = event.params.time;
-
+  request.ancillaryData = event.params.ancillaryData.toHex();
   request.isGovernance = event.params.isGovernance;
+
+  // If governance request we manually create an identifier as it doesn't exist on chain
+  if (event.params.isGovernance) {
```
Comment (md0x) [inline packages/votingV2/src/mappings/votingV2.ts:54]: If it's a governance price request I decided to manually create an Identifier to be able to query it in the subgraph. Does it make sense to you @chrismaree ?

File: packages/votingV2/abis/VotingV2.json [hunk 2]
```diff
@@ -1,745 +1,263 @@
...
-        "name": "cumulativeActiveStake",
-        "type": "uint256"
-      },
-      {
-        "indexed": false,
-        "internalType": "uint256",
-        "name": "cumulativePendingStake",
-        "type": "uint256"
-      }
+      { "indexed": true, "internalType": "address", "name": "voter", "type": "address" },
```
Comment (chrismaree) [inline packages/votingV2/abis/VotingV2.json:133]: yay for linting!

File: packages/votingV2/src/utils/constants.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { toDecimal } from "./decimals";
 export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
 export const STORE_ID = "0x54f44ea3d2e7aa0ac089c4d8f7c93c27844057bf";
 export const GOVERNOR_ADDRESS_STRING = "0x592349f7dedb2b75f9d4f194d4b7c16d82e507dc";
-export let VOTING_TOKEN_ADDRESS = Address.fromString("0x04fa0d235c4abf4bcf4787af4cf447de572ef828");
+export let VOTING_TOKEN_ADDRESS = Address.fromString("0xBa950974AFdae520cEE596c98295150Cf8D9659c");
```
Comment (chrismaree) [inline packages/votingV2/src/utils/constants.ts:7]: it seems kinda strange that we store these here vs a const file with all the addresses in one place. I know this was a carry over from the previous implementation but we should consider cleaning this up a bit.

File: packages/votingV2/manifest/data/goerli.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
   "VotingDataSources": [
     {
       "name": "Voting",
-      "address": "0xf21d2478F75D405b6D1e368D313D06B8AD20E088",
+      "address": "0x11680368E6a4F83Db1855629782f7b78F470fF62",
       "startBlock": 7311557
```
Comment (chrismaree) [inline packages/votingV2/manifest/data/goerli.json:14]: this should also be updated, right?

**Review by chrismaree — APPROVED** [review]

**Pull request merged**